k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -4402,6 +4395,25 @@ public boolean modeMismatch(Account.FolderMode aMode, Folder.FolderClass fMode)
         }
     }
 
+    private boolean isContact(Address[] addrs) {
```

Comment:
That should definitly be changed, I just didn't see that method. The only thing left is the foreach-loop, should I move that to shouldNotifyForMessage or add a method in Contacts which takes an Address[]?